src/components/expandableSection/index.tsx

```diff
@@ -0,0 +1,43 @@
+import _ from 'lodash';
+import React from 'react';
+import {LayoutAnimation, StyleSheet} from 'react-native';
+import View from '../view';
+
+export type ExpandableSectionProps = {
+  /**
+   * expandableSection header element
+   */
+  sectionHeader?: JSX.Element;
+  /**
+   * expandableSection expandable children
+   */
+  children?: React.ReactNode;
+  /**
+   * expandableSection icon color
+   */
+  expanded?: boolean;
+};
+
+function ExpandableSection(props: ExpandableSectionProps) {
+  const {expanded, sectionHeader, children} = props;
+
+  function onValueChange() {
+    LayoutAnimation.configureNext({...LayoutAnimation.Presets.easeInEaseOut, duration: 200});
```

Comment:
We don't trigger animation or update things during a render. 
Render can be triggered for many reasons and that means you'll trigger an animation even when you don't intend to. 
For that we use hooks like `useEffect`. (or lifecycle methods in case you use classes)